### PR TITLE
feat: Support textarea

### PR DESCRIPTION
**Pull request by yanggun7201**
I just want to support also `textarea` so that I add `inputType` to support it.
It should be one of the `input` or `textarea`.

### CODE REVIEW

File: src/components/ReactTags.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ class ReactTags extends Component {
     ),
     allowUnique: PropTypes.bool,
     renderSuggestion: PropTypes.func,
+    inputType: PropTypes.oneOf(["input", "textarea"]),
```
Comment (ad1992) [inline src/components/ReactTags.js]: use `singleQuote`

File: src/components/ReactTags.js [hunk 2]
```diff
@@ -90,6 +91,7 @@ class ReactTags extends Component {
     allowUnique: true,
     allowDragDrop: true,
     tags: [],
+    inputType: "input",
```
Comment (ad1992) [inline src/components/ReactTags.js]: use `singleQuote`

File: src/components/ReactTags.js (hunk 1 above)
Comment (yanggun7201) [reply src/components/ReactTags.js]: I fixed it. thanks

File: src/components/ReactTags.js (hunk 2 above)
Comment (yanggun7201) [reply src/components/ReactTags.js]: I fixed it. thanks

**Comment by yanggun7201** [comment]
> @yanggun7201 thanks for your contribution, could you make the changes and also add spec for this change

Can I just change `README.md` file? or more files?
Could you please let me know about it?
Thanks.

**Comment by ad1992** [comment]
> > @yanggun7201 thanks for your contribution, could you make the changes and also add spec for this change
> 
> Can I just change `README.md` file? or more files?
> Could you please let me know about it?
> Thanks.

@yanggun7201 can you add a spec for this in `ReactTags.spec.js` file

**Pull request closed without merging**